src/dispatch/__tests__/Dispatch.test.ts

```diff
@@ -479,36 +479,6 @@ describe('Dispatch tests', () => {
         expect((dispatch as unknown as any).enabled).toBe(true);
     });
 
-    test('when a fetch request is rejected with 401 then dispatch is disabled', async () => {
```

Comment:
nit: do we need reverse of this unit test?

Reply:
It's not scalable for every status code to have a separate unit test. In the future, we should consolidate them into either "keep alive" or "disable".